src/content/nft/index.md

```diff
@@ -108,7 +108,15 @@ NFT's have some special properties:
 In other words, if you _own_ an NFT:
 
 - You can easily prove you own it.
-  - Proving you own an NFT is very similar to proving you have ETH in your account
+  - Proving you own an NFT is very similar to proving you have ETH in your account.
+  - For example, lets say you pay for an NFT. What you are getting is the right to transfer your token to your digital wallet.
+  - The token proves that your copy of the digital file is the original.
+  - Your private crypto key is proof of ownership of the original.
+  - The content creator's public crypto key serves as a certificate of authenticity for that particular digital artifact.
+    - This public and private key pair is what primarily determines the value of any NFT token.
+  - Another way to think about proving you own the NFT is by signing messages to prove you own the private key behind the address.
+    - As mentioned above, your private key is proof of ownership of the original. Now this in turn tells us that the NFT is controlled by private keys behind that address.
```

Comment:
(similarly, I believe "proof-of-ownership" should be hyphenated when in the middle of a sentence)